app/seeders/util/util.cjs

```diff
@@ -26,4 +29,19 @@ async function bulkUpsert(
   }
 }
 
-module.exports = { bulkUpsert };
+async function parseFile(folder, filename) {
+  const records = [];
+  const parser = fs
+    .createReadStream(
+      `${__dirname}/../../seed-data/${folder}/${filename}.csv`,
```

Comment:
This feels fragile!

Reply:
`__dirname` is always the directory name of the current file. So unless util.cjs is moved, this works fine (at which point it can be adjusted). I don't really want to handle absolute file paths here, since they will be different on every machine. So this seems like the best approach to me.